responsibleai/requirements.txt

```diff
@@ -7,6 +7,7 @@ numpy>=1.17.2
 pandas>=0.25.1
 scikit-learn>=0.22.1
 scipy>=1.4.1
+semver~=2.13.0
```

Comment:
Looks like this is only used in test, not product code? If so, shouldn't it be in requirements-dev?

Reply:
My original intention was to do full power semver ranging checking in the package code, not just the tests. Looks like I stopped short of that, will keep this comment open until I add that

Reply:
Added semver checking